### PR TITLE
Add docs for Consul Ent on ECS

### PR DESCRIPTION
**Pull request by lkysow**

### CODE REVIEW

File: website/content/docs/ecs/enterprise.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+| Sentinel                          | Yes (servers) | This feature runs on Consul servers.                                                                                                                                    |
+| Network Segments                  | No            | Currently there is no capability to configure the network segment Consul clients on ECS run in.                                                                         |
+| Namespaces                        | No            | Currently there is no capability to configure the Consul namespace for a service on ECS.                                                                                |
+| Admin Partitions                  | No*           | Supported if Consul ECS clients run in the default partition. Otherwise there is currently no capability to configure the admin partition Consul clients in ECS run in. |
+| Audit Logging                     | No*           | Audit logging can be enabled on Consul servers that run outside of ECS but is not currently supported on the Consul clients that run inside ECS.                        |
+
```
Comment (pglass) [inline website/content/docs/ecs/enterprise.mdx:65]: 🎉

**Review by pglass — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/505190.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 3c687662861def0582be0d523e1f29ca8957e238 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 3c687662861def0582be0d523e1f29ca8957e238 onto `release/1.10.x` succeeded!

**Pull request merged**